src/docs/timestamp-prepare.dox

```diff
@@ -50,4 +50,22 @@ transactions should set the durable and commit timestamps to the same time.
 Prepared transactions have their own configuration keyword for rounding
 timestamps; see @ref timestamp_roundup for more information.
 
+@section timestamp_prepare_ignore_prepare Configuring ignore_prepare
+
+The WT_SESSION::begin_transaction method includes the \c ignore_prepare
+configuration. The \c ignore_prepare configuration causes readers to
+ignore prepared transactional values, that is, returning read values as if
+the prepared transaction didn't exist. This prevents readers from seeing
+the ::WT_PREPARE_CONFLICT error, returning the data as it was before the
+transaction was prepared. Setting the \c ignore_prepare configuration to
+true also causes the transaction to be read-only, and attempts to update
+items in the transaction will fail.
```

Comment:
Is it worth also stating explicitly that `ignore_prepare` means that applications can't rely on repeatable read semantics in a transaction, since the same read at a later time (after the prepared transaction is resolved) could lead to a different value being returned?

Reply:
Good point -- fixed in 3fff7f436.